chaosaws/ec2/actions.py

```diff
@@ -119,8 +131,12 @@ def terminate_instance(instance_id: str = None, az: str = None,
                              'set of filters')
 
     if az and not any([instance_id, filters]):
-        logger.warning(
-            'Configuration will terminate a random instance in %s.' % az)
+        authorize = get_user_input(
```

Comment:
Oh hey, I see what you did here.

Maybe we should have discussed it beforehand. I think your approach stands to reason but we are trying to not surprise users with specific (per-driver or per-action) workflows.

So, I'm not going to allow that here unfortunately on that basis.
We should discuss in the issue I linked about a global approach to this question.

Or if you feel this makes this action too dangerous, then we should remove it entirely for now. 
I can live with a warning but since it won't stop the experiment, it may not help anyway.

Reply:
Ok, I'll revert back to warnings... will add some thoughts to the issue you linked earlier.